src/screens/ChatScreen/components/CannedResponses.js

```diff
@@ -67,20 +82,23 @@ const CannedResponses = ({ cannedResponses, onClick }) => {
   const theme = useTheme();
   const styles = useMemo(() => createStyles(theme), [theme]);
   return (
-    <View style={styles.mainView}>
-      <FlatList
-        data={cannedResponses}
-        renderItem={({ item, index }) => (
-          <CannedResponse
-            shortCode={item.shortCode}
-            content={item.content}
-            lastItem={cannedResponses.length - 1 === index}
-            onClick={onClick}
-          />
-        )}
-        keyExtractor={item => item.id}
-        keyboardShouldPersistTaps={'handled'}
-      />
+    <View style={[cannedResponses.length > 0 && styles.mainView]}>
+      {cannedResponses.length > 0 && (
```

Comment:
Create a flag variable and use it in JSX.
`const isCannnedResponsesExist = cannedResponses.length > 0 `

Reply:
Done